src/main.rs

```diff
@@ -15,12 +17,53 @@ async fn main() {
     let rust_lang_org = octocrab.orgs("rust-lang");
     let repos: Vec<Repository> = all_repos(&&rust_lang_org).await?;
 
+    println!("#PR,\tREPO\n--------------------");
     for repo in &repos {
-        println!("repo: {}", repo.name);
+        let count_prs = count_pull_requests(&octocrab, &repo.name).await?;
+        println!("{},\t{}", count_prs, repo.name);
     }
 }
 
 #[throws]
 async fn all_repos(org: &octocrab::orgs::OrgHandler<'_>) -> Vec<octocrab::models::Repository> {
     util::accumulate_pages(|page| org.list_repos().page(page).send()).await?
 }
+
+#[throws]
+async fn count_pull_requests(octo: &octocrab::Octocrab, repo_name: &String) -> usize {
+    let init_page = octo
+        .pulls("rust-lang", repo_name)
+        .list()
+        .sort(octocrab::params::pulls::Sort::Created)
+        .per_page(100)
+        .send()
+        .await?;
+
+    let thirty_days_ago = Utc::now() - Duration::days(30);
+    let mut pr_count: usize = init_page.items.len();
```

Comment:
bug! this assumes all PRs in the initial page are, in fact, within the last 30 days.

I'll fix this.

Reply:
fixed in 943f301